.agents/skills/aiperf-pr-checkout/SKILL.md

```
@@ -0,0 +1,98 @@
+
+For correctness review (`aiperf-code-review`), ensure `origin/main` is up to date:
+
+```bash
+git fetch origin main
+```
+
+### 5. Report back
+
+When done, report to the calling skill:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)